gbpservice/nfp/service_plugins/vpn/drivers/nfp_vpnaas_driver.py

```diff
@@ -29,6 +31,9 @@
 
 BASE_VPN_VERSION = '1.0'
 AGENT_TYPE_VPN = 'NFP Vpn agent'
+ACTIVE = 'ACTIVE'
+ERROR = 'ERROR'
+TIMEOUT = 45
```

Comment:
Waiting for 45 seconds between VPN service create and IPSec connection create is too long as configurator is not processing anything. Let us keep this to 30 sec.